packages/module/src/Message/UserFeedback/UserFeedbackComplete.test.tsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import UserFeedbackComplete from './UserFeedbackComplete';
+
+describe('UserFeedbackComplete', () => {
+  it('should render correctly', () => {
+    render(<UserFeedbackComplete timestamp="12/12/12" />);
+    expect(screen.getByText('Thank you')).toBeTruthy();
+    screen.getByText(/You have successfully sent your feedback!/i);
```

Comment:
```suggestion
    screen.getByText(/We've received your response. Thank you for sharing your feedback!/i);
```

Reply:
I'd like to suggest changing the thank-you message to be: 

> Feedback submitted
> We've received your response. Thank you for sharing your feedback!

I see a number of references to the current text, so figured it might be easier for you to search and replace vs me comment on each line!